README.md

```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
                      'ga': 0.21617801281707663,
                      'start': 0.13906813268582238}}
 
+## Install
+
+### Requirements
+
+* Python 3.5, 3.6 or 3.7
+* All the system requirements of [revscoring](https://github.com/wikimedia/revscoring)
+
+### Installation steps
+
+1. clone this repository
+2. install its dependencies `python -m pip -r requirements.txt`
+3. install the package itself `python setup.py install`
+4. Now you can run `make` to build the english wikipedia article quality model or `make wikidatawiki_models` to build the item quality model for wikidata
```

Comment:
This isn't really an installation step.  #4 should be moved to a section called "building models" or something like that.

Reply:
Oh also, when you run "make", it should build all of the models -- not just the English Wikipedia one.

Reply:
Yes, I intended them more as a way to verify that the installation worked. I'll update the wording to reflect that